pkg/controller/packages/pkg/pkg.go

```diff
@@ -856,9 +856,13 @@ func (h *packageHandler) prepareDeployment(d *apps.Deployment) {
 	d.Spec.Template.Spec.ServiceAccountName = h.ext.Name
 
 	if !h.allowFullDeployment {
-		d.Spec.Template.Spec.SecurityContext = &corev1.PodSecurityContext{
+		psc := &corev1.PodSecurityContext{
 			RunAsNonRoot: &runAsNonRoot,
 		}
+		if d.Spec.Template.Spec.SecurityContext != nil && d.Spec.Template.Spec.SecurityContext.FSGroup != nil && *d.Spec.Template.Spec.SecurityContext.FSGroup != 0 {
```

Comment:
nit: there was great discussion in the PR around why/how we landed on blocking fsgroup 0 here also.  consider including a brief summary as a comment so that decision/policy lives with the code as well.

Reply:
Sounds good, will update :+1:

Reply:
@jbw976 this is now updated. If you wouldn't mind taking a look at the note and making sure that it matches your understanding as well that would be much appreciated :)